src/modules/voxelformat/MeshExporter.cpp

```diff
@@ -30,7 +30,17 @@ bool MeshExporter::saveGroups(const SceneGraph& sceneGraph, const core::String &
 	const bool mergeQuads = core::Var::get(cfg::VoxformatMergequads, "true", core::CV_NOPERSIST)->boolVal();
 	const bool reuseVertices = core::Var::get(cfg::VoxformatReusevertices, "true", core::CV_NOPERSIST)->boolVal();
 	const bool ambientOcclusion = core::Var::get(cfg::VoxformatAmbientocclusion, "false", core::CV_NOPERSIST)->boolVal();
+
 	const float scale = core::Var::get(cfg::VoxformatScale, "1.0", core::CV_NOPERSIST)->floatVal();
+
+	float scaleX = core::Var::get(cfg::VoxformatScaleX, "1.0", core::CV_NOPERSIST)->floatVal();
```

Comment:
Here `core::Var::getSafe` should be used (unlike the others here - they are still doing it wrong, too) - the cvars should be created with `core::Var::get` in the `construct()` methods. See e.g. the places where e.g. `cfg::VoxformatMergequads` is created - (`VoxConvert::construct()` and `SceneManager::construct()`)

this allows use to show the cvars in `./vengi-voxconvert --help`

The documentation should get updated to list those new cvars, too (see Formats.md)

Reply:
OK